common/src/main/java/org/jboss/hal/testsuite/page/config/DeploymentScannerPage.java

```diff
@@ -0,0 +1,33 @@
+package org.jboss.hal.testsuite.page.config;
+
+import org.jboss.arquillian.graphene.Graphene;
+import org.jboss.hal.testsuite.finder.Application;
+import org.jboss.hal.testsuite.finder.FinderNames;
+import org.jboss.hal.testsuite.finder.FinderNavigation;
+import org.jboss.hal.testsuite.fragment.ConfigFragment;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Created by pcyprian on 22.10.15.
+ */
+public class DeploymentScannerPage extends ConfigurationPage {
+
+    public void navigateToDeploymentScanners() {
+        FinderNavigation navigation = new FinderNavigation(browser, StandaloneConfigEntryPoint.class)
+                .addAddress(FinderNames.CONFIGURATION, FinderNames.SUBSYSTEMS)
+                .addAddress(FinderNames.SUBSYSTEM, "Deployment Scanners");
+
+        navigation.selectRow().invoke("View");
+        Application.waitUntilVisible();
+    }
+
+    public ConfigFragment getWindowFragment() {
+        WebElement editPanel = browser.findElement(By.className("default-window-content"));
+        return  Graphene.createPageFragment(ConfigFragment.class, editPanel);
+    }
+
+    public ConfigFragment getMainCongifFragment() {
```

Comment:
My last request: please rename `getMainCongifFragment` to `getMainConfigFragment` ;-)